lib/howitzer/email.rb

```diff
@@ -55,8 +63,10 @@ def self.adapter=(adapter_name)
     # * +recepient+ - recepient's email address
     #
 
-    def self.find_by_recipient(recipient)
-      find(recipient, self::SUBJECT)
+    def self.find_by_recipient(recipient, *args)
+      raise NoSubjectError, "Please specify email subject. For example:
+                             class SomeEmail < Howitzer::Email\n subject ‘some subject text’\nend" if args.nil?
```

Comment:
extra spaces here, it is possible to split long string via / symbol. For example:

"Hello " /
"World"